mm2src/mm2_main/src/lp_swap/swap_watcher.rs

```diff
@@ -25,8 +25,8 @@ struct WatcherContext {
     ctx: MmArc,
```

Comment:
Related to `const WAIT_FOR_TAKER_REFUND: u64 = 1200; // How long?`: I think it can be even half of taker payment lock duration.

Reply:
You mean, the watcher should refund the taker payment after watcher_ctx.data.swap_started_at + watcher_ctx.data.lock_duration + (watcher_ctx.data.lock_duration / 2), is that correct?

Reply:
Yeah, half of the lock duration should be safe to do so.